### DISCUSSION TITLE
How to create PDF/Latex Documentation without the Text: "It follows a list of topics" ?

### DISCUSSION THREAD

**Discussion by paule32** [doxygen/doxygen#11202] [Q&A] [answered]
Hello,
like the Subject says: how can I remove the Text:
"Es folgt eine Liste der Themen..."

Translated: "It follows a list of topics".

![grafik](https://github.com/user-attachments/assets/63db1a4a-5294-4441-9584-55e1e908cfe4)

**Comment by albert-github** [comment, marked as answer]
The text is steered through the `intro` attribute of the `topics`tab in the doxygen layout file.
By replacing the line:
```
<tab type="topics" visible="yes" title="" intro=""/>
```
by
```
<tab type="topics" visible="yes" title="" intro=" "/>
```

one can accomplish the desired effect.

In the image a second question is given, it is unclear what is meant here.

Note when giving a text it would be better not tho use the `OUTPUT_LANGUAGE=German` but the default English version so the original translation would have been used (`Here is a list of all topics with brief descriptions:`).
